packages/gp2-components/src/organisms/UserNavigation.tsx

```diff
@@ -1,19 +1,39 @@
 import {
+  Avatar,
+  chevronDownIcon,
+  chevronUpIcon,
   colorWithTransparency,
   drawerQuery,
   navigationGrey,
   paper,
   steel,
   tin,
-  UserMenuButton,
+  pixels,
 } from '@asap-hub/react-components';
 import { useCurrentUserGP2 } from '@asap-hub/react-context';
 import { css } from '@emotion/react';
 import { ComponentProps, useEffect, useRef, useState } from 'react';
-import { smallDesktopQuery } from '../layout';
 
 import UserMenu from '../molecules/UserMenu';
 
+const { rem } = pixels;
+
+const buttonResetStyles = css({
+  width: rem(80),
+  padding: 0,
+  backgroundColor: 'unset',
+  border: 'none',
+  outline: 'none',
+});
+const styles = css({
+  padding: `${rem(12)} 0`,
+  cursor: 'pointer',
+
+  display: 'grid',
+  gridTemplateColumns: `auto ${rem(48)}em auto auto`,
```

Comment:
`${rem(48)}em`
did you mean to have that `em`?
`rem` function already adds `em` so I'm thinking it will maybe end up being `<some_value>emem`, no?